index.js

```diff
@@ -0,0 +1,76 @@
+document.addEventListener("DOMContentLoaded",()=>{
+
+fetch("https://ghibliapi.herokuapp.com/films")
+.then(response => response.json())
+.then(response => {
+  getFilms(response)
+})
+
+let select = document.querySelector("select")
+select.addEventListener("change",(e)=>{
```

Comment:
Listener functions when long is better to take to a function definition outside instead of defining it in the arguments.

Reply:
It would have been good to break down the listener for `change` in the select into function of the like: `getMovieDetails()` and `renderMovieDetails()`. It would have also been good to put the initial fetch request to get all the movies into a function like `getAllMovies()` and just call the function in the listener for `DOMContentLoaded`.